### ISSUE TITLE
Add classification section

### ISSUE THREAD

**Issue by lorensr** [apollographql/apollo-feature-requests#38]
README currently has broken "classify" link to:

https://github.com/apollographql/apollo-client/CONTRIBUTING.md#classification

**Comment by lorensr** [comment]
We're also missing the `confirmed` label

**Comment by hwillson** [comment]
Thanks @lorensr - I've removed the classify link completely. We're still thinking about the best FR classification approach moving forward, and we'll get the `README` updated shortly.

Regarding the `confirmed` label missing, I'm wondering if we can come up with a better label. `confirmed` in the context of a feature request means that the feature request has been fully spec'd out, and is ready to be worked on. But `confirmed` in my mind can be a bit misleading - when dealing with bugs, `confirmed` makes a lot of sense if the bug has been confirmed (by running a reproduction for example). When dealing with a FR though, `confirmed` seems out of place - does it mean we've confirmed this issue is a feature request, or we've confirmed this is something that should be looked into further, or we've confirmed it's ready to be worked on, etc.?

I'm thinking instead of `confirmed` we go with something like `ready`. So `ready` means the FR is fully spec'd, and it's ready to be implemented. Thoughts?

**Comment by lorensr** [comment]
Agreed! I like ready. That's if we want to differentiate between features
that are ready to be implemented and features that also have a good chance
of being accepted (which also get the help-wanted label).

Would be simpler to have one category, and only put something into it when
it's both clear and something that would go in apollo-client vs a package.
And try to merge all such PRs. In which case help-wanted feels clearer to
be than ready.
